base/osutils.jl

```diff
@@ -1,20 +1,26 @@
 # This file is a part of Julia. License is MIT: http://julialang.org/license
 
-"""
-    OS_NAME
+let UNAME = ccall(:jl_get_UNAME, Any, ())
+    # change some `uname -s` values to "friendly" names
+    # for internal usage
```

Comment:
If the goal of midipix is run Linux applications on Windows, WSL does a better job. midipix has failed in that regard, in that they're not measurably getting closer to anything we can use. The Linux binaries of Julia do actually run in WSL though. It doesn't make the win32 build of Julia any less necessary because it doesn't interoperate and can't run on older than Windows 10, but this is beside the point. The value of this variable is not actually the kernel, so it shouldn't be called that.